gen-api-reference/README.md

```diff
@@ -38,3 +38,40 @@ at the root of the repo, and that your GOPATH is set properly. For example:
 1. Run `git diff` to verify the changes.
 
 1. Create a PR to merge your changes.
+
+# Generating the Pipelines API Reference Page
+
+These are instructions for generating a copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/) page from the swagger file. 
+
+## Prerequisites
+* Download the [pipelines](https://github.com/kubeflow/pipelines) repository.
+* Install [npm](https://www.npmjs.com/get-npm).
+
+## Installation
+
+1. Extract the pipelines repository to a local directory.
```

Comment:
This seems like a duplicate of the prerequisite step to clone the repository. Is there a compressed file that needs to be extracted as well?

Reply:
Yes, specifically the swagger file is the only piece actually needed to perform the process. I have revised this direction and others to better call that out.